src/plotting/plot_mysql_dir_sensor_sat.py

```diff
@@ -0,0 +1,224 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas , matplotlib.pyplot as plt
+from datetime import datetime, date
+import matplotlib.dates as mdates
+from IPython.display import display
+import os
+from scipy import interpolate
+import argparse
+import glob
+import obs_inv_utils.inventory_table_factory as itf
+import re
+
+#argparse section
+parser = argparse.ArgumentParser()
+parser.add_argument("-o", dest='out_dir', help="output directory for figures",default='figures',type=str)
+parser.add_argument("--sidb", dest='satinfo_db_root', help="root for sat info db files",default='satellites/satinfo/',type=str)
+parser.add_argument("-dev", dest='dev', help='Use this flag to add a timestamp to the filename for development', default=False, type=bool)
+args = parser.parse_args()
+
+#parameters
+satinfo_db_root=args.satinfo_db_root
+daterange=[date(1975,1,1), date(2025,1,1)]
+
+sat_dictionary={"NOAA 5": "n05", "NOAA 6": "n06", "NOAA 7": "n07", "NOAA 8": "n08", "NOAA 9": "n09", 
```

Comment:
this is now moved to utils. remove here.

Reply:
updated